starboard/elf_loader/exported_symbols.cc

```diff
@@ -432,6 +433,7 @@ ExportedSymbols::ExportedSymbols() {
   REGISTER_SYMBOL(calloc);
   REGISTER_SYMBOL(close);
   REGISTER_SYMBOL(connect);
+  REGISTER_SYMBOL(errno);
```

Comment:
Use `__errno_location` instead and exclude `src/starboard/errno/__errno_location.c` from the BUILD.gn file for `musl`.